maud/Cargo.toml

```
@@ -16,7 +16,7 @@ maud_htmlescape = { version = "0.17.0", path = "../maud_htmlescape" }
 maud_macros = { version = "0.20.0", path = "../maud_macros" }
 iron = { version = ">= 0.5.1, < 0.7.0", optional = true }
 rocket = { version = ">= 0.3, < 0.5", optional = true }
-actix-web = { version = ">= 0.6.12, < 0.8.0", optional = true }
+actix-web = { version = "1.0.0", optional = true, default-features=false }
```

Would be good to add spaces around the `=` for consistency

```suggestion
actix-web = { version = "1.0.0", optional = true, default-features = false }
```